src/app/home-page/home-page.component.ts

```diff
@@ -8,6 +8,7 @@ import { ProjectSummary } from '../shared/models/project-summary';
   styleUrls: ['./home-page.component.css']
 })
 export class HomePageComponent implements OnInit {
+  title = "My Projects";
```

Comment:
Do you have TSLint set up in your IDE? Our tslint.json wants 'single' quotemarks